### PR TITLE
xUnit math tests

### PR DESCRIPTION
**Pull request by Frassle**
### Purpose of this PR

* Provide a template for xUnit based math tests written in C#.
* This is to encourage more PR's from the community which is primarily C# not F# based.

### CODE REVIEW

File: tests/OpenTK.Tests.Math/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -0,0 +1,22 @@
+﻿// <auto-generated/>
```
Comment (Nihlus) [inline tests/OpenTK.Tests.Math/Properties/AssemblyInfo.cs]: This shouldn't be in source control.

Comment (Frassle) [reply tests/OpenTK.Tests.Math/Properties/AssemblyInfo.cs]: Yes I know, I had it removed in another commit in the fixing paket PR.
Seems I haven't quite got paket happy with this right now either. Missing an xunit reference.

Comment (ThomasD13) [reply tests/OpenTK.Tests.Math/Properties/AssemblyInfo.cs]: Great!! I will start to transfer my code to xUnit.

**Comment by Frassle** [comment]
> Very nice. Could you port over the arbitrary generators and some examples of how to use those as well?

Yes but I want this basic template in ASAP so @ThomasD13 can base his Quaternion tests off it. I'll get the generators done ASAP as another PR based on this.

**Comment by Nihlus** [comment]
LGTM

**Pull request merged**